src/main/kotlin/com/terraformation/backend/search/SearchFilter.kt

```diff
@@ -126,6 +130,9 @@ data class FieldNode(
   override fun toString(): String {
     return "FieldNode($field $type [${values.joinToString()}])"
   }
+
+  private val isFuzzySearchForNull: Boolean
+    get() = type == SearchFilterType.Fuzzy && values.any { it == null }
```

Comment:
should we consider an empty values list as search for null?

Reply:
Empty values lists aren't legal in the API.

Reply:
Though I guess right now that's only enforced in the controller, and there also ought to be a check for it in the implementation. Will follow up with a separate PR for that.

Reply:
Got it 👍